gen/builders/builder.go

```diff
@@ -100,6 +100,14 @@ func MessageVector(metadata *Metadata) *Builder {
 	return b
 }
 
+// Selector sets a selector string on the vector.
+func (b *Builder) Selector(selector string) {
+	if selector == "" {
+		return
```

Comment:
You cannot unset a selector?

Reply:
I don't think you'd want to unset a selector that you've just set for a vector?